uint16 cmd_wd
---
string cmd_stat